changelog/66200.fixed

```diff
@@ -0,0 +1 @@
+'Fixed documentation about unique id behavior' 
```

Comment:
```suggestion
Fixed documentation about unique id behavior
```

Reply:
That's odd. I don't remember typing those quotes. I guess I copied pasted it from the shell or something. Alyssa said we don't want change log files for documentation changes so I will push a commit without it.